src/presentation/components/FormLoaderStatus/index.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { RenderResult, render } from '@testing-library/react';
+import { FormLoaderStatus } from '@/presentation/components';
+import Context from '@/presentation/hooks/form';
+
+const renderComponent = (isLoading: boolean, mainError: string): RenderResult => {
```

Comment:
I recommend to use the name "makeSut" to this const, due to the pattern in the other test scenarios

Reply:
Done